lib/utils/analytics.js

```diff
@@ -1,11 +1,19 @@
 /* eslint-disable no-undef */
 import { ANALYTICS_GROUPS } from 'lib/utils/analyticsConstants';
 
-const sendDataToAnalytics = ({ action, category, label, value, non_interaction = false }) => {
+const sendDataToAnalytics = ({
+  action,
+  category,
+  label,
+  value,
+  custom_text,
+  non_interaction = false
+}) => {
   gtag('event', action, {
     event_category: category,
     event_label: label,
     value,
+    custom_text,
```

Comment:
Do the rest of analytics still work the same?

Reply:
Yep, I've tested the other's are still sending data. This is a custom dimension that won't show in the default screens / reports. If someone inspected the other events in a custom report and added this as a dimension, it would show as '(not set)'. This was tested against Staging/UA where you can see this new data and existing values being logged.